From: Andrea Pappacoda <tachi@debian.org>
Date: Sun, 23 Mar 2025 15:53:37 +0100
X-Dgit-Generated: 0~20250314+ds-4 dcdc63ee7b516868fb12c06006c95d0d0da8f65b
Subject: 51-android.rules: use plugdev group instead of adbusers

Forwarded: not-needed

---

diff --git a/51-android.rules b/51-android.rules
index d0d6e49..aa5a5a7 100644
--- a/51-android.rules
+++ b/51-android.rules
@@ -1098,7 +1098,7 @@ ENV{adb_ptp}=="yes", ENV{adb_user}="yes", ATTR{bDeviceClass}=="00|02|06|ef|ff",
 ENV{adb_mtp}=="yes", ENV{adb_user}="yes", SYMLINK+="libmtp-%k", ENV{ID_MTP_DEVICE}="1", ENV{ID_MEDIA_PLAYER}="1"
 
 # Enable device as a user device if found (add an "android" SYMLINK)
-ENV{adb_user}=="yes", MODE="0660", GROUP="adbusers", TAG+="uaccess", SYMLINK+="android", SYMLINK+="android%n"
+ENV{adb_user}=="yes", MODE="0660", GROUP="plugdev", TAG+="uaccess", SYMLINK+="android", SYMLINK+="android%n"
 
 # Devices listed here {begin...end} are connected by USB
 LABEL="android_usb_rules_end"
